benchmarks/dlrm/requirements_repo.in

```diff
@@ -0,0 +1,10 @@
+future
+numpy
+onnx
+pydot
+torch
+torchviz
+scikit-learn
+tqdm
+torchrec-nightly
+torchx-nightly
```

Comment:
Does it really need nightly?

Reply:
From their requirements yes but it's possible that it was prior the release of pt 2.0